certify-service/src/test/java/io/mosip/certify/controller/VCIssuanceControllerTest.java

```diff
@@ -70,15 +70,15 @@ public void getIssuerMetadata_withQueryParam_thenPass() throws Exception {
         Map<String, Object> issuerMetadata = new HashMap<>();
         issuerMetadata.put("credential_issuer", "https://localhost:9090");
         issuerMetadata.put("credential_endpoint", "https://localhost:9090/v1/certify/issuance/credential");
-        issuerMetadata.put("credentials_supported", Arrays.asList());
+        issuerMetadata.put("credential_supported", Arrays.asList());
 
         Mockito.when(vcIssuanceService.getCredentialIssuerMetadata(Mockito.anyString())).thenReturn(issuerMetadata);
 
         mockMvc.perform(get("/issuance/.well-known/openid-credential-issuer?version=v11"))
                 .andExpect(status().isOk())
                 .andExpect(jsonPath("$.credential_issuer").exists())
                 .andExpect(jsonPath("$.credential_issuer").exists())
-                .andExpect(jsonPath("$.credentials_supported").exists())
+                .andExpect(jsonPath("$.credential_supported").exists())
```

Comment:
Is this correct ? In draft 11 was it called credential_supported without the s ?

Reply:
I've updated it